index.html

```diff
@@ -116,25 +116,33 @@
         profile in an unambiguous way. This mechanism can be
         used to define additional profiles.
       </li>
-      <li>In addition it defines a <strong><a>Core
+      <li>It define a first base <strong><a>Core
             Profile</a></strong> of the Thing Description for use with
-        the HTTP(S) and selected notification protocols. The <a>Core
-          Profile</a> formalizes the results of several PlugFests
-        that were conducted by the WoT Interest Group and of
-        tests that were conducted as part of the development. It
-        is expected that additional profiles for thing templates
-        and other protocols will be defined in the near future.
+        the HTTP(S) and selected sub-protocols (e.g., for eventing). 
```

Comment:
Not sure we should talk / or select subprotocols. Current proposed event mechanisms include SSE and Longpoll.
These just use HTTP.

Reply:
SSE and Longpoll are sub-protocols of HTTP. This is aligned with the current TD spec. Please also see https://www.w3.org/TR/wot-thing-description/#form